supabase/migrations/44_draft_associated_collections.sql

```diff
@@ -0,0 +1,74 @@
+begin;
+
+-- Update permissions for live_spec_flows to allow users to directly access it as long as they
+-- have access to at least one side. This may allow users to know the id of specifications that
+-- they are not authorized to, but nothing else.
+alter policy "Users must be authorized to referenced specifications" on live_spec_flows
+  rename to "Users must be authorized to one referenced specification";
+
+alter policy "Users must be authorized to one referenced specification" on live_spec_flows
+  using (
+    source_id in (select id from live_specs) or
+    target_id in (select id from live_specs)
+  );
+grant select on live_spec_flows to authenticated;
+
+create or replace function get_collections_eligible_for_deletion(capture_id flowid)
```

Comment:
Can this be inlined directly into `draft_collections_eligible_for_deletion()`, which seems like a pretty reasonable API? Then we don't have to take a position on whether this too should be public API and can refactor it later.

Or is it actively being used by the UI in the proposed feature?

Reply:
It's not used. I think it came from a desire to show what _would_ be deleted, but that's not a thing that the UI does currently. I removed it.